From: Dan Williams <dan.j.williams@intel.com>
Date: Tue, 10 Apr 2018 14:23:47 -0700
Subject: tools/testing/nvdimm: fix missing newline in nfit_test_dimm 'handle'
 attribute
Patch-mainline: v4.17-rc2
Git-commit: 19357a685e870eeb825cbb7fd3104082ab041987
References: bsc#112128

Sysfs userspace tooling generally expects the kernel to emit a newlines
when reading sysfs attributes.

Signed-off-by: Dan Williams <dan.j.williams@intel.com>
Acked-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 tools/testing/nvdimm/test/nfit.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/tools/testing/nvdimm/test/nfit.c
+++ b/tools/testing/nvdimm/test/nfit.c
@@ -1194,7 +1194,7 @@ static ssize_t handle_show(struct device
 	if (dimm < 0)
 		return dimm;
 
-	return sprintf(buf, "%#x", handle[dimm]);
+	return sprintf(buf, "%#x\n", handle[dimm]);
 }
 DEVICE_ATTR_RO(handle);
 
